contracts/TermsContract.sol

```diff
@@ -65,12 +48,10 @@ interface TermsContract {
         uint256 timestamp
     ) public view returns (uint256);
 
-     /// Returns the cumulative units-of-value repaid by the point at which a given block timestamp has lapsed.
+     /// Returns the cumulative units-of-value repaid by the point at which this method is called
      /// @param  agreementId bytes32. The agreement id (issuance hash) of the debt agreement to which this pertains.
-     /// @param timestamp uint. The timestamp of the block for which repayment value is being queried.
-     /// @return uint256 The cumulative units-of-value repaid by the time the given timestamp lapsed.
+     /// @return uint256 The cumulative units-of-value repaid up until now.
     function getValueRepaid(
```

Comment:
I would consider calling this function `getValueRepaidToDate` to be more explicit that it reflects all payments made up until now.